src/extensions/install/install.cmd.tsx

```diff
@@ -21,7 +21,12 @@ export default class InstallCmd implements Command {
   // TODO: remove this ts-ignore
   // @ts-ignore
   async render() {
-    const results = await this.install.install();
-    return <Color green>Successfully installed {results.length} component(s)</Color>;
+    try {
+      const results = await this.install.install();
+      return <Color green>Successfully installed {results.length} component(s)</Color>;
+    } catch (e) {
```

Comment:
don't handle it yourself here, throw a `PaperError`

Reply:
I agree. I had issues with it though so did this temporarily. Will fix in this or a future PR in the next few days as I refactor things